recipes-bsp/u-boot/u-boot-orangepi-i96.bb

```diff
@@ -0,0 +1,35 @@
+require ${COREBASE}/meta/recipes-bsp/u-boot/u-boot.inc
+
+FILESEXTRAPATHS_prepend := "${THISDIR}/u-boot:"
```

Comment:
we don't apply any patches or ship extra files, so the line can be removed entirely.